test/autowaiting.spec.js

```diff
@@ -196,7 +196,7 @@ describe('Auto waiting should not hang when', () => {
         setTimeout(() => window.stop(), 100);
       }, server.EMPTY_PAGE);
   });
-  it.fail(CHROMIUM)('calling window.stop sync', async({page, server, httpsServer}) => {
+  it('calling window.stop sync', async({page, server, httpsServer}) => {
     // Flaky, see https://github.com/microsoft/playwright/pull/1630/checks?check_run_id=553475173.
```

Comment:
nit: please remove this comment. Also, reading the comment, I am not sure how does this patch address the issue :)

Reply:
Done